twoliter/embedded/Makefile.toml

```diff
@@ -1398,7 +1410,6 @@ pubsys \
 # Rather than depend on "build", which currently rebuilds images each run, we
 # depend on publish-tools and check for the input file below to save time.
 # This does mean that `cargo make ami` must be run before `cargo make validate-ami`.
-dependencies = ["fetch-sources"]
```

Comment:
Seems like we can get rid of most of these 'Rather than depend on "build"' comments now, since these tasks don't depend on "publish-tools" or anything else.